datafusion/sqllogictest/test_files/expr.slt

```diff
@@ -1487,32 +1487,32 @@ SELECT extract(epoch from arrow_cast('1969-12-31', 'Date64'))
 
 # test_extract_interval
 
-query R
+query I
 SELECT extract(year from arrow_cast('10 years', 'Interval(YearMonth)'))
 ----
 10
 
-query R
+query I
```

Comment:
Should we have explicit tests inspecting `arrow_typeof(extract(....))`?

Reply:
Since `I` is for integer and we don't need to make sure it is exactly i32 or i64 so I think that is fine. If we want to differentiate type like utf8 or utf8View then we need `arrow_typeof`

Reply:
> Since `I` is for integer

yes i know



> need to make sure it is exactly i32 or i64

do we? do we not?
i think it may make difference, since it's 2x fewer bytes to keep in memory

up to you